Build instructions don't work on Ubuntu 18.04.3 LTS

## Description

On Ubuntu 18.04.3 LTS, I followed the steps for Debian build from the README, and got this message during CMake. The problem is the Botan version is 2.4.0 but RNP 0.12 requires 2.5.0.

This relates to #980.

## Steps to Reproduce

1. `git clone https://github.com/rnpgp/rnp -b v0.12.0`
2. `mkdir rnpbuild; cd rnpbuild`
3. `cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=on -DBUILD_TESTING=off ../rnp/`

## Expected Behavior

RNP compiles.

## Actual Behavior

```
HEAD is now at 586caec6 Release 0.12.0
root@aa16642f9338:/rnp# cd ../rnpbuild/
root@aa16642f9338:/rnpbuild# cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=on -DBUILD_TESTING=off ../rnp/
-- RNP_VERSION: 0.12.0
-- RNP_VERSION_NCOMMITS: 0
-- RNP_VERSION_GIT_REV: 586caec
-- RNP_VERSION_IS_DIRTY: FALSE
-- RNP_VERSION_COMMIT_TIMESTAMP: 0
-- RNP_VERSION_SUFFIX: 
-- RNP_VERSION_FULL: 0.12.0
CMake Error at /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find Botan2: Found unsuitable version "2.4.0", but required is at
  least "2.5.0" (found /usr/lib/libbotan-2.so)
Call Stack (most recent call first):
  /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:376 (_FPHSA_FAILURE_MESSAGE)
  cmake/Modules/FindBotan2.cmake:91 (find_package_handle_standard_args)
  src/lib/CMakeLists.txt:33 (find_package)
```

--- a/docs/installation.adoc
+++ b/docs/installation.adoc
@@ -22,6 +22,35 @@ curl -L https://github.com/riboseinc/yum/raw/master/ribose.repo > /etc/yum.repos
 yum install -y rnp
 ----
 
+== On Ubuntu 18
+
+[source,console]
+----
+# Clone the repository by version tag (or omit it to get the latest sources)
+sudo apt install git
+git clone https://github.com/rnpgp/rnp.git -b v0.12.0
+
+# Install required packages
+sudo apt install g++-8 cmake libbz2-dev zlib1g-dev libjson-c-dev build-essential python-minimal
+
+# Download, build and install Botan2
+wget -qO- https://botan.randombit.net/releases/Botan-2.12.1.tar.xz | tar xvJ
+cd Botan-2.12.1
+./configure.py --prefix=/usr
+make
+sudo make install
+cd ..
+
+# Cmake recommend out-of-source builds
+mkdir rnp-build
+cd rnp-build
+
+# Cmake it
+cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=on -DBUILD_TESTING=off ../rnp/
+make
+sudo make install
+----
+
 == On Debian
 
 [source,console]

